### PR TITLE
debug: forward systemd journald output to serial console when booting for uc20

### PR DESCRIPTION
**Pull request by anonymouse64**
This will hopefully make the serial log we get for nested uc20 runs more clear
about when the reboots happen to see if there's some common thread between them.

Also comment out a couple unit tests which break when we do this (as they 
should), as otherwise we won't get spread runs on GitHub from the PR.

Also make the tests/nested/core20/basic test always fail so we can see the logs from github actions.

### CODE REVIEW

**Comment by anonymouse64** [comment]
One such log from a run I triggered locally: https://pastebin.ubuntu.com/p/7ftJpJJbzt/

File: tests/nested/core20/basic/task.yaml [hunk 1]
```diff
@@ -31,3 +31,6 @@ execute: |
 
     echo "Ensure 'snap list' works and test-snapd-sh snap is removed"
     not execute_remote "snap list test-snapd-sh"
+
+    # make us fail so we get the boot logs for random reboot debugging
```
Comment (sergiocazzolato) [inline tests/nested/core20/basic/task.yaml:35]: if you run spread -shell google-nested:ubuntu-20.04-64:tests/nested/core20/basic then you can see the logs without exit 1

Comment (anonymouse64) [reply tests/nested/core20/basic/task.yaml:35]: right but you can't do that for a PR, I guess you could if you changed run-checks but that seemed like more work than I wanted to :-)

**Comment by anonymouse64** [comment]
this PR served it's purpose but is no longer needed

**Pull request closed without merging**